unittests/common.h

```diff
@@ -0,0 +1,27 @@
+/*
+  Copyright (C) 2011 - 2018 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_unittests_common_h
+#define _aspect_unittests_common_h
+
+#include <catch.hpp>
```

Comment:
Why not just `#include` this in all the places where we do `#include <common.h>` right now?

Reply:
You are right in that it is kind of useless right now, but I want to put extra utilities to be used in tests in this file in the very near future.

Reply:
OK